sunbird-platform-core/actor-core/pom.xml

```diff
@@ -25,17 +25,17 @@
 		<dependency>
 			<groupId>com.typesafe.akka</groupId>
 			<artifactId>akka-actor_2.11</artifactId>
-			<version>2.5.16</version>
+			<version>2.5.19</version>
```

Comment:
same as above . use variable to point 2.5.19 version

Reply:
done